guides/source/association_basics.md

```diff
@@ -832,7 +832,7 @@ The `belongs_to` association supports these options:
 * `:inverse_of`
 * `:polymorphic`
 * `:touch`
-* `:validate`
+* `:optional`
```

Comment:
We still have the `:validate` option. It is not the same as `:required` so we should keep its documentation.